tests/services/test_text_processor.py

```
@@ -33,9 +33,8 @@ def mock_text_model() -> MagicMock:
 @pytest.fixture
 def text_processor(mock_text_model: MagicMock) -> TextProcessor:
     """TextProcessor instance with a mocked model."""
-    with patch("services.text_processor.ensure_nltk_data"):
-        processor = TextProcessor(text_model=mock_text_model)
-        return processor
+    processor = TextProcessor(text_model=mock_text_model)
+    return processor
```

Ruff (0.15.10)

[warning] Unnecessary assignment to `processor` before `return` statement

Remove unnecessary assignment

(RET504)